Amplify/DevMenu/View/DeviceInfoRow.swift

```diff
@@ -7,6 +7,7 @@
 
 import SwiftUI
 
+/// View corresponding to each row in Device Information Screen
 @available(iOS 13.0.0, *)
 struct DeviceInfoRow: View {
```

Comment:
I wonder if `*InfoView` could be more generic to accommodate both Device and Environment info.